src/components/pages/pricing/hero/hero.jsx

```diff
@@ -23,7 +23,7 @@ const items = [
     description: 'Generous always-available free tier, no credit card required.',
     features: [
       { title: '0.5 GiB storage' },
-      { title: '24/7 for your main compute', info: 'Plus 20h of usage for branches' },
+      { title: '24/7 for your primary compute', info: 'Plus 20h of usage for branches' },
```

Comment:
`default`?

Reply:
We're now going to have "Primary" and "Read Replica" computes, as per: https://github.com/neondatabase/cloud/issues/14509